website/data/plugins-nav-data.json

```
@@ -65,6 +65,10 @@
           {
             "title": "Windows IIS",
             "path": "drivers/community/iis"
+          },
+          {
+            "title": "Wasmtime",
+            "path": "drivers/community/wasmtime"
           }
```

These should be in alphabetical order:

```suggestion
          {
            "title": "Wasmtime",
            "path": "drivers/community/wasmtime"
          },
          {
            "title": "Windows IIS",
            "path": "drivers/community/iis"
          }
```